hypervisor/kvmtool/lkvm.go

```diff
@@ -439,6 +439,10 @@ func (kc *KvmtoolContext) Save(ctx *hypervisor.VmContext, path string) error {
 }
 
 func (kc *KvmtoolContext) ConnectConsole(console chan<- string) error {
+	if kc.conPty == "" {
```

Comment:
This check if ok, and we should implement Save interface for kvmtool, so the conPty will not be nil.

Reply:
While reading this code, I have some doubt about calling watchConsole in Launch and VmAssociate.
I am not sure why we have to open VM console device twice?

Reply:
I don't know if I get your point correctly, is this pr https://github.com/hyperhq/runv/pull/624 relate to your doubt?

Reply:
Yes, that PR#624 is relate to my doubt. Thanks Feng!